crosspoint_reader/driver.py

```
@@ -274,6 +276,48 @@ def _ensure_dir(self, parent_path, subdirs):
+            converter.convert_epub(input_path, temp_path)
+            
+            return temp_path
+            
+        except Exception as exc:
+            self._log(f'[CrossPoint] Conversion failed: {exc}')
+            # Clean up temp file on failure
+            if temp_path:
+                try:
+                    os.remove(temp_path)
+                except Exception:
+                    pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)